src/app.module.ts

```diff
@@ -7,10 +7,13 @@ import { ConfigModule } from '@nestjs/config';
 import { PrismaModule } from '../prisma/prisma.module';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { config } from './config/config';
 
 @Module({
   imports: [
-    ConfigModule.forRoot(),
+    ConfigModule.forRoot({
+      validationSchema: config
+    }),
```

Comment:
đã custom file config thì sẽ cần load nó vào AppModule, ở đây em chưa có app configuration file nhé
```suggestion
    ConfigModule.forRoot({
      validationSchema: config
    }),
```